app/models/game.rb

```diff
@@ -1,4 +1,72 @@
 class Game < ActiveRecord::Base
-
 	belongs_to :user
+  has_many :pieces
+
+  after_create :populate_board!
+
+  def populate_board!
+    # Store the game's id in a var. Note: .pluck returns an array
+    game_id = Game.pluck(:id)
+
+    # These are initialized to the white pieces' y coordinates.
+    # Note: y2 is for the pawns' y coordinate.
+    y = 1
+    y2 = 2
+
+    # Create white pieces locations on 1st loop and then 
+    # the black pieces locations on the 2nd loop
+    2.times do
+      # Create 8 pawns                   
+      1.upto(8) do |x|
+        Pawn.create( :x_position => x,
+                     :y_position => y2,
+                     :active     => 1,
+                     :game_id    => game_id[0] )
```

Comment:
:white_check_mark: Done. @mscoutermarsh I modified your suggested code and used `game => self.id`. Worked like a charm. Thanks for that solution.
![woot-baby](https://cloud.githubusercontent.com/assets/3744711/9049943/8d72ee58-39e3-11e5-9230-1c3f135fe4bc.jpg)